components/FloorMenu.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect } from "react";
+import { StyleSheet, AsyncStorage, View } from "react-native";
+import SwitchSelector from "react-native-switch-selector";
+
+const hallFloors = [
+    { label: "1", value: "1" },
+    { label: "2", value: "2" },
+    { label: "3", value: "3" },
+    { label: "4", value: "4" },
+    { label: "5", value: "5" },
+    { label: "6", value: "5" },
+    { label: "7", value: "7" },
+    { label: "8", value: "8" },
+    { label: "9", value: "9" },
+    { label: "10", value: "10" },
+    { label: "11", value: "11" },
+    { label: "12", value: "12" },
+    { label: "13", value: "13" },
+];
+
+const jmsbFloors = [
+    { label: "1", value: "1" },
+    { label: "2", value: "2" },
+    { label: "3", value: "3" },
+    { label: "5", value: "4" },
+    { label: "6", value: "5" },
+    { label: "7", value: "7" },
+    { label: "8", value: "8" },
+    { label: "9", value: "9" },
+    { label: "10", value: "10" },
+    { label: "11", value: "11" },
+    { label: "12", value: "12" },
+    { label: "13", value: "13" },
+    { label: "14", value: "14" },
+    { label: "15", value: "15" },
+];
+
+export function FloorMenu() {
+    const [floorNumber, setFloorNumber] = React.useState("1");
+    const [selectedBuilding, setSelectedBuilding] = React.useState("");
+    AsyncStorage.setItem("floorSelected", floorNumber);
+
+    const buildingSelected = async () => {
```

Comment:
Same comment here. **buildingSelected** and **selectedBuilding** need a distinction :(